render.c

```diff
@@ -86,15 +84,15 @@ static int render_notification(cairo_t *cairo, struct mako_state *state,
 	// Render border
 	set_source_u32(cairo, config->colors.border);
 	cairo_set_line_width(cairo, border_size * scale);
-	set_rectangle(cairo, 0, offset_y, state->width, notif_height, scale);
+	set_rectangle(cairo, 0, offset_y, config->width, notif_height, scale);
```

Comment:
Hmm, this still needs to be `state`, because even if the compositor gave us less width than we asked, we still want to render ourselves correctly.

Reply:
OK, I wasn't sure about that. I'll pass both.